posthog/temporal/data_imports/external_data_job.py

```diff
@@ -150,6 +179,14 @@ def parse_inputs(inputs: list[str]) -> ExternalDataWorkflowInputs:
     async def run(self, inputs: ExternalDataWorkflowInputs):
         assert inputs.external_data_schema_id is not None
 
+        if settings.TEMPORAL_TASK_QUEUE == DATA_WAREHOUSE_TASK_QUEUE_V2:
```

Comment:
I'm confused what the workflow scheme is here

If a worker's env var is task queue v2 it should be picking up the workflows being kicked off in line 143 but then it will hit this line above again and end up in an infinite loop of firing and picking up these jobs

Reply:
Ahh, this is my bad - this should be a `!=` - I refactored the env var and forgot to update here